semgrep/semgrep/settings.py

```diff
@@ -53,7 +53,7 @@ def __init__(self) -> None:
     @staticmethod
     def get_path_to_settings() -> Path:
         """
-        Uses ~/.semgrep/settings.yaml unless SEMGREP_SETTINGS_FILE is set
+        Uses {$XDG_CONFIG_HOME/semgrep || ~/.semgrep}/settings.yaml unless SEMGREP_SETTINGS_FILE is set
```

Comment:
I'm not in love with my update to this docstring -- open to feedback here

Reply:
Seems fine :D